src/Rule/RegexHandler.php

```diff
@@ -42,13 +36,7 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
             (!$rule->isNot() && !preg_match($rule->getPattern(), $value)) ||
             ($rule->isNot() && preg_match($rule->getPattern(), $value))
         ) {
-            $result->addError(
-                $rule->getMessage(),
-                [
-                    'attribute' => $context->getAttribute(),
-                    'value' => $value,
```

Comment:
`value` can be useful for custom messages.

Reply:
https://github.com/yiisoft/validator/pull/352#discussion_r1006452279

Reply:
https://github.com/yiisoft/validator/pull/352#discussion_r1006471638